src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class FigureSupplier extends ColorSupplier {
+    public static final int NUMBER_OF_FIGURES_NAME = 5;

```

Comment:
The constant `NUMBER_OF_FIGURES_NAME` is slightly misleading. Consider renaming it to `FIGURE_COUNT` or `NUMBER_OF_FIGURES` to better reflect its purpose.